apps/backend/src/modules/users/user.service.ts

```diff
@@ -93,6 +94,22 @@ class UserService implements Service {
 		return await this.userRepository.findPasswordDetails(email);
 	}
 
+	public async findUserProfile(
```

Comment:
Where is the tests for all new logic on backend?
```suggestion
public async findUserProfile(
```